counter.rb

```diff
@@ -0,0 +1,28 @@
+class Counter
+  def initialize(filename)
+    @filename = filename
+  end
+
+  # Do everything
+  def run
+    contents = File.read(@filename)
+    word_hash = split_text(contents)
+    print_descending(word_hash)
+  end
+
+  # Splits the text into a hash of words and counts
```

Comment:
using `private` here is probably a good idea. it signals to other devs that the methods below are "dependencies."  (i.e. internal methods that we wouldn't test.)